scalability_and_performance/enabling-workload-partitioning.adoc

```diff
@@ -6,57 +6,34 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-In resource-constrained environments, you can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs.
+Workload partitioning segregates particular workloads into distinct CPU sets. The primary objective is ensure that the platform pods will not exceed a specified CPU usage limit. You can use workload partitioning to isolate {product-title} services, cluster management workloads, and infrastructure pods to run on a reserved set of CPUs. This ensures that the remaining CPUs in their deployment remain untouched and available exclusively for their own workloads. The minimum number of reserved CPUs required for the cluster management is four CPU Hyper-Threads (HTs).
```

Comment:
It's not the workloads that get segregated but rather the compute node cpu resources.

And the objective is not to ensure that platform pods don't exceed usage limits but rather that system tasks are restricted to specific CPUs only. And also what you mention below, that the cpu capacity of the host is correctly accounted for